maker/src/main.rs

```diff
@@ -191,11 +191,13 @@ async fn main() -> Result<()> {
     let auth_username = rocket_basicauth::Username("itchysats");
     let auth_password = seed.derive_auth_password::<rocket_basicauth::Password>();
 
-    let (identity_pk, identity_sk) = seed.derive_identity();
+    let keypair = seed.derive_identities();
 
-    let hex_pk = hex::encode(identity_pk.to_bytes());
+    let peer_id = keypair.libp2p.public().to_peer_id();
```

Comment:
🙃 Could also hide the way we access the internals and just have `identities.peer_id()` here.

Reply:
great suggestion!